.vscode/settings.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
@emmanuelmathot Could you remove this file since it's specific to your IDE?

Reply:
removed